zanata-war/src/main/java/org/zanata/limits/RateLimitManager.java

```diff
@@ -125,7 +125,7 @@ public String getFieldValueAsString(String fieldName) {
                 });
     }
 
-    public RestCallLimiter getLimiter(final String apiKey) {
+    public RestCallLimiter getLimiter(final String key) {
```

Comment:
> Can be 
1) username(if no api key generate for that user)
2) api key
3) anonymous IP address.

Of course 1) case will not happen as the request will already be rejected in ZanataRestSecurityInterceptor

I do think it's important to distinguish between these cases.  What about a simple immutable value class with factory methods which take either username, API key or IP address?

Also, won't case 1 happen when we invoke REST services from the browser, if the user has no api key?